helper/include/odp/helper/cli.h

```diff
@@ -131,6 +132,20 @@ int odph_cli_run(void);
  */
 int odph_cli_stop(void);
 
+/**
+ * Print to CLI
+ *
+ * Similar to odph_cli_log(), except that this one takes its arguments as
+ * a va_list.
+ *
+ * @param fmt printf-style message format
+ * @param in_args variadic arguments
+ * @return On success, the number of characters printed or buffered, without
+ *         accounting for any line feed conversions. If an error is encountered,
+ *         a negative value is returned.
+ */
+int odph_cli_log_va(const char *fmt, va_list in_args);
```

Comment:
Nitpicking, but it would be more logical that `odph_cli_log()` is defined before `odph_cli_log_va()`.